ra_aid/proc/interactive.py

```
@@ -1,90 +1,216 @@
+
+    captured_data = []
+    start_time = time.time()
+    was_terminated = False
+
+    def check_timeout():
+        elapsed = time.time() - start_time
+        if elapsed > 3 * expected_runtime_seconds:
+            # Force kill after 3x expected runtime
+            os.killpg(os.getpgid(proc.pid), signal.SIGKILL)
+            return True
+        elif elapsed > 2 * expected_runtime_seconds:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)